css-text-decor-4/Overview.bs

```diff
@@ -160,6 +160,22 @@ Line Decoration: Underline, Overline, and Strike-Through</h2>
 		represents the span element, and the orange lines represent the
 		blocks.
 	</div>
+	<div class="example">
+		In the following style sheet and document fragment:
+		<pre>
+		  div { color: black; font-size: 48px; text-decoration: underline; text-shadow: blue 0px 50px 0px; }
+		  span { font-size: 20px; vertical-align: top; text-shadow: green 0px 100px 0px; }
+		</pre>
+		<pre>
+                  &lt;div&gt;Help, help! &lt;span&gt;I am under a hat!&lt;/span&gt;&lt;/div&gt;
+		</pre>
+		...the &lt;div&gt; is the decorating box, so the black underline is rendered uninterrupted through
+                both the &lt;div&gt; and the &lt;span&gt;. Decorations are painted as part of 'text-shadow's, but
+                'text-shadow' is inherited, unlike 'text-decoration-line'. Therefore, the green text-shadow on the
+                &lt;span&gt; clobbers the blue text-shadow on the &lt;div&gt;, so the shadow's underline is disjoint
+                across the two elements.
```

Comment:
Suggested rephrasing for clarity (also fixing tabs vs spaces and wrapping per [semantic line break convention](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)):

```suggestion
		...the &lt;div&gt; is the [=decorating box=] for its underline (in black),
		which is rendered uninterrupted through both the &lt;div&gt; and the &lt;span&gt;.
		Unlike line decorations, however,
		`text-shadow` is inherited as a property;
		therefore the green text shadow on the &lt;span&gt;
		overrides the blue text shadow on the &lt;div&gt;.
		As a result, when the shadows are painted,
		the shadow of the &lt;div&gt;’s underline is disjoint across the two elements.
```